client/src/storage/remote-storage.ts

```diff
@@ -20,8 +25,8 @@ export class RemoteStorage implements IStorage {
             })
     }
 
-    public async getIslandData(islandName: string): Promise<IslandData> {
-        return this.fetchResource(`/island/${islandName}`)
+    public async getIslandData(islandName: string): Promise<IslandData | undefined> {
```

Comment:
Currently, if island isn't defined i return a default value {hexalots: "", spots: ""}. I should change it to just return a 404 instead

Reply:
(this is a note to self)

Reply:
I guess that assumes the server already has known islands.  I'm tempted to choose the 30 largest cities in the world.

Reply:
...and Rotterdam :) 

Yeah, there's an ENABLED_ISLANDS constant on the server that we can populate with these cities.

Reply:
```
export enum Islands {
    Tokyo,
    Delhi,
    Shanghai,
    SaoPaulo,
    MexicoCity,
    Cairo,
    Mumbai,
    Beijing,
    Dhaka,
    Osaka,
    Karachi,
    BuenosAires,
    Chongqing,
    Istanbul,
    Kolkata,
    Manila,
    Lagos,
    RioDeJaneiro,
    Tianjin,
    Guangzhou,
    Moscow,
    Shenzen,
    Lahore,
    Bangalore,
    Paris,
    Bogota,
    Jakarta,
    Chennai,
    Lima,
    Bangkok,
}

```

Reply:
Sweet! I'mma include this in a PR

Reply:
I pushed this on the server side.